Frontend/ui-customer/src/Flow.purs

```diff
@@ -3868,4 +3850,68 @@ updateEmergencySettings state = do
                 }
               , data { settingSideBar { hasCompletedSafetySetup = true } }
               }
-        )
\ No newline at end of file
+        )
+  
+checkForSpecialZoneAndHotSpots :: HomeScreenState -> ServiceabilityRes -> Number -> Number -> FlowBT String Unit
```

Comment:
Can we move these functions to transformer instead of flow

Reply:
@ashishsingh101

Reply:
we are not transforming instead returning FlowBT so keeping it in flow itself but moved others to transformer